go/pools/rp_bench_test.go

```diff
@@ -137,35 +137,3 @@ func BenchmarkGetPutMixedMulti(b *testing.B) {
 		}
 	}
 }
-
-func BenchmarkApplySettings(b *testing.B) {
-	b.ReportAllocs()
-	ctx := context.Background()
```

Comment:
Any reason why the Benchmark is removed? Do we not need it any more?

Reply:
earlier this benchmark shows how executing a single query over multiple query have better effect, but with the current changes there will always be only single query that will be received. making this benchmark unnecessary.